spec/jobs/voice_otp_sender_job_spec.rb

```diff
@@ -19,12 +22,28 @@
       expect(call.to).to eq('555-5555')
       expect(call.from).to match(/(\+19999999999|\+12222222222)/)
       expect(call.url).to include('1234')
+      expect(call.record).to eq(true)
+    end
+
+    context 'when twilio_record_voice is true' do
```

Comment:
should we also have another context that explicitly sets `expect(Figaro.env).to receive(:twilio_record_voice).and_return('true')`?